pkg/model/bootstrapscript.go

```diff
@@ -511,3 +514,27 @@ func (b *BootstrapScript) createProxyEnv(ps *kops.EgressProxySpec) string {
 	}
 	return buffer.String()
 }
+
+func usesOpenStack(cluster *kops.Cluster) bool {
+	if kops.CloudProviderID(cluster.Spec.CloudProvider) == kops.CloudProviderOpenstack {
```

Comment:
@olemarkus Why isn't this enough to decide that the cloud provider is OpenStack?

Reply:
Technically you could run on AWS, but use swift as configbase. No one is probably doing this though.
The use case is common though, if you openstack as provider, using S3 as configbase is common.

Reply:
You are using this function to check if the cluster is OS.
Why is it not enough to check `cluster.Spec.CloudProvider`?